src/components/ThankYou/ThankYou.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./ThankYou.scss";
+import { BigNumber } from "./../BigNumber/BigNumber"
+
+export const ThankYou = (props) => {
+    return (
+        <div className="ThankYou">
+            <h2 className="ThankYou__text">{props.thanksText}</h2>
+            <h3 className="ThankYou__orderText">{props.orderText}</h3>
+            <div className="ThankYou__numberWrap">
+                <BigNumber
+                text={props.numberText}
+                />
+            </div>
+            <div className="ThankYou__deliveryTitle">{props.deliveryTitle}</div>
```

Comment:
We got this wrong, it is Barlow and the alignment is wrong to, It's centered, but aligned to the RIGHT, its not centered and also centered with the text. Also your barlow 500 wieght is not working